Source/UnitTests/Common/MathUtilTest.cpp

```diff
@@ -64,8 +64,8 @@ TEST(MathUtil, SaturatingCast)
   EXPECT_EQ(std::numeric_limits<s32>::min(),
             MathUtil::SaturatingCast<s32>(-std::numeric_limits<float>::infinity()));
   // 16777217 = 2^24 + 1 is the first integer that cannot be represented correctly with a f32.
-  EXPECT_EQ(16777216, MathUtil::SaturatingCast<s32>(float(16777216)));
-  EXPECT_EQ(16777216, MathUtil::SaturatingCast<s32>(float(16777217)));
```

Comment:
I think this is a perfect example of `static_cast` providing no benefit and only hurting readability.
I do not see a benefit of changing functional style casts of non-pointer types (like `float` or `u64`) to `static_cast`.
The type of the cast in `u8(expression)` or `float(numbers)` is perfectly clear and typing out `static_cast` is overly verbose for no reason.